tests/test_debug/test_snapshot_manager.py

```diff
@@ -0,0 +1,151 @@
+from __future__ import annotations
+
+import os.path
+import shutil
+import sys
+
+import pytest
+from pytest_snapshot.plugin import Snapshot
+
+from python_ta.debug import SnapshotManager
+
+SNAPSHOT_DIR = os.path.join(
+    os.path.dirname(os.path.realpath(__file__)), "snapshot_manager_testing_snapshots"
+)
+TEST_RESULTS_DIR = "/tmp/test_results"
+
+
+def func_one_line(output_directory: str = None) -> None:
```

Comment:
This file is pretty lengthy with multiple types of definitions (functions that are the "inputs" to the test cases, fixtures, helpers, the actual test cases). You can improve the organization by doing the following:

1. Organize the functions so that they are organized into sections (no mixing of types) and give section a comment as a section header.
2. Write the test cases at the top, then the function inputs, then the helpers, then the fixtures.